src/client/components/Appheader/AppHeader.component.js

```diff
@@ -0,0 +1,37 @@
+// working with dumb/statless component with no props
+
+import React from 'react';
+import './AppHeader.style.css'; // css for the div not for the img
+import Vector from './Vector.svg'; // css and propreties come directly within the svg file
+
+export const AppHeader = () => {
+  return (
+    <div className="AppHeader-div">
+      <img src={Vector} alt="AppHeader" />
+    </div>
+  );
+};
+
+/*
+
+// dumb/statless img component with props
```

Comment:
here comments should remove .

Reply:
They are related to the other component. check the whole file because I wrote the same component in 2 different ways. one is active and the other is commented